src/Storages/MergeTree/DataPartStorageOnDisk.h

```diff
@@ -149,6 +149,12 @@ class DataPartStorageBuilderOnDisk final : public IDataPartStorageBuilder
         size_t buf_size,
         const WriteSettings & settings) override;
 
+    std::unique_ptr<WriteBufferFromFileBase> writeFile(
```

Comment:
Not a big issue but perhaps we can stick with one `writeFile()` call which is passed the write mode as default parameter.

Reply:
The code seems not ours... leave it as is.